src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
+    that you can supply the configuration values to the individual components of the chart, as listed at the beginning of this section, by prefixing them with the name of the chart. 
+    
+    For example, to run the agent in
+    unprivileged mode, you can add `--set newrelic-infrastructure.privileged=false` to the command above, where
+    `newrelic-infrastructure` is the name of the sub-chart (as listed at the beginning of the section), and `privileged`
+    is the configuration value for said chart.
+
+    Please refer to [each of the chart's README](https://github.com/newrelic/helm-charts/tree/master/charts/) to see all
+    available configuration options.
+
   </Collapser>
 
   <Collapser
-    className="freq-link"
-    id="install-helm2"
-    title="Helm 2"
+      className="freq-link"
+      id="install-helm2"
+      title="Helm 2 using newrelic-infrastructure"
   >
     To install using Helm 2:
 
     1. Set the cluster where you want to install the agent:
 
-       ```
-       kubectl config set-cluster <var>DESIRED_CLUSTER</var>
-       ```
-
-       To see the available clusters, run `kubectl config get-clusters`
-    2. If your cluster uses role-based access, create a service account for `tiller` (Helm's service which runs inside the Kubernetes cluster) using a Kubernetes manifest file:
-
-       1. Paste the following in a manifest file `rbac-config.yaml`:
-
-          ```
-          apiVersion: v1 
-          kind: ServiceAccount 
-          metadata: 
-            name: tiller 
-            namespace: kube-system 
-          --- 
-          apiVersion: rbac.authorization.k8s.io/v1 
-          kind: ClusterRoleBinding 
-          metadata: 
-            name: tiller 
-          roleRef: 
-            apiGroup: rbac.authorization.k8s.io 
-            kind: ClusterRole 
-            name: cluster-admin 
-          subjects: 
-            - kind: ServiceAccount 
-              name: tiller 
-              namespace: kube-system
-          ```
-       2. Create the service account and role by running:
-
-          ```
-          kubectl create -f rbac-config.yaml
-          ```
+    ```
+    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
+    ```
+
+    To see the available clusters, run `kubectl config get-clusters`
+
+    2. Make sure that `kube-state-metrics` is installed on your machine:
+
+    ```
+    kubectl get deployment --all-namespaces | grep kube-state-metrics
+    ```
+
+    If it's not installed, follow the [instructions in the kube-state-metrics GitHub
+    repo](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment) to install it.
+
+    3. If your cluster uses role-based access, create a service account for `tiller` (Helm's service which runs inside
+    the Kubernetes cluster) using a Kubernetes manifest file:
+
+    4. Paste the following in a manifest file `rbac-config.yaml`:
```

Comment:
Should we point to https://v2.helm.sh/docs/rbac/#example-service-account-with-cluster-admin-role ?

Reply:
Done

Reply:
I have stripped the whole section and pointed to the v2 quickstart as a reference. I don't think it belongs here.